src/ape/api/transactions.py

```diff
@@ -295,6 +295,26 @@ def await_confirmations(self) -> "ReceiptAPI":
 
         return self
 
+    @property
+    def return_value(self) -> Any:
+        """
+        Obtain
```

Comment:
```suggestion
        Obtain the final return value of the call. Requires tracing to function,
        since this is not available from the receipt object.
        
        Returns:
            Any: The return value (ABI Decoded) of the call
```